### PR TITLE
Ironic: increasing the default nodes listed 

### PR DESCRIPTION
**Pull request by lekhajee**

### CODE REVIEW

File: mimic/model/ironic_objects.py [hunk 1]
```diff
@@ -219,15 +219,15 @@ def list_nodes(self, include_details):
         http://docs.openstack.org/developer/ironic/webapi/v1.html#get--v1-nodes-detail
         """
         if not self.ironic_node_store:
-            for each in range(3):
+            for each in range(30):
```
Comment (ksheedlo) [inline mimic/model/ironic_objects.py]: style nit: if `each` is not used in the body of the loop, `_` can be used as a throwaway iterator variable. Helps express that the loop is run purely for the side effect.

File: mimic/test/test_ironic.py [hunk 2]
```diff
@@ -84,9 +84,9 @@ def test_list_nodes_with_details(self):
         instance_nodes = [node['extra']['flavor'] if node['extra'].get('flavor')
                           else None
                           for node in content['nodes']]
-        self.assertEqual(instance_nodes.count('onmetal-io1'), 5)
-        self.assertEqual(instance_nodes.count('onmetal-compute1'), 3)
-        self.assertEqual(instance_nodes.count('onmetal-memory1'), 3)
+        self.assertEqual(instance_nodes.count('onmetal-io1'), 32)
```
Comment (ksheedlo) [inline mimic/test/test_ironic.py]: Noticing that this is 2 more than the default - where do those come from?

Comment (ksheedlo) [inline mimic/test/test_ironic.py]: Nevermind - these are the 2 without the configured flavor at the end.

**Comment by ksheedlo** [comment]
LGTM :+1:

**Comment by lekhajee** [comment]
@ksheedlo : thanks for the review

**Pull request merged**